backend/actions/event.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import { Event } from "../models/Event";
+import mongoose from "mongoose";
```

Comment:
Importing dependency should look clear.

Sort the dependencies.
1. Put the external dependencies first and sort them.
2. Put a line break.
3. Put the internal dependencies and sort them.